packages/grafana-ui/src/components/Button/Button.story.tsx

```diff
@@ -36,27 +37,29 @@ export default {
 
 export const Examples: StoryFn<typeof Button> = () => {
   return (
-    <VerticalGroup>
+    <Stack direction="column">
       {allButtonFills.map((buttonFill) => (
-        <VerticalGroup key={buttonFill}>
-          <HorizontalGroup spacing="lg">
+        <Stack direction="column" key={buttonFill}>
+          <Stack gap={3}>
             {allButtonVariants.map((variant) => (
-              <VerticalGroup spacing="lg" key={`${buttonFill}-${variant}`}>
+              <Stack direction="column" gap={3} key={`${buttonFill}-${variant}`}>
                 {sizes.map((size) => (
-                  <Button variant={variant} fill={buttonFill} size={size} key={size}>
-                    {variant} {size}
-                  </Button>
+                  <Box key={size}>
```

Comment:
I'm not sure this `Box` is needed, if the problem is how the `Button`s get aligned we can just change the `alignItems` of the parent `Stack`?

Reply:
Nope, the problem was that those buttons behaved like full-width buttons, instead of maintaining their size.

Reply:
If you remove the `Box` and add `alignItems="flex-start"` to the parent it looks the same as the old story 🤔

Reply:
Yay! True! Thanks! 🙌